packages/modelviewer.dev/examples/loading/index.html

```diff
@@ -30,7 +30,7 @@
   <script src="../../node_modules/focus-visible/dist/focus-visible.js" defer></script>
 
   <script>
-    window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
```

Comment:
I tried reverting this, but not sure why I still see the changes :/ 
git checkout origin/master ./packages/modelviewer.dev/index.html

Reply:
I think you're looking for `git checkout origin/master -- ./packages/modelviewer.dev/index.html` a subtle but important distinction.

Reply:
I tried that too, didn't work :/